system/jlib/jptree.cpp

```diff
@@ -8638,7 +8638,7 @@ Owned<IPropertyTree> getGlobalConfigSP()
     return getGlobalConfig();
 }
 
-jlib_decl IPropertyTree * loadArgsIntoConfiguration(IPropertyTree *config, const char * * argv, std::initializer_list<const char *> ignoreOptions)
+jlib_decl IPropertyTree * loadArgsIntoConfiguration(IPropertyTree *config, const char * * argv, std::initializer_list<const std::string> ignoreOptions)
```

Comment:
This change doesn't look like it is related - and I think it will mean previously constexpr arrays now need to dynamically initialise the object array.

Reply:
It's fixing a bug I found while testing, that the suppression of certain flags when using this function does not work as the comparison is done using pointers rather than strcmp. There's only one place that passes in an initializer array, and it compiles on on my machine/branch

Reply:
It should compile, just slightly less efficient.  Given the context it is called in, and the number of times it is used, that is probably irrelevant.